app/views/posts/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>Here is a list of posts for a given user</h1>
+<section class="container">
+        <header>
+            <%=render 'shared/user',user:@user%>
+        </header>
+        <div>
+                <%@posts.each do |post|%>
+                    <div class="my-5">
+                        <%=render 'shared/post',post:post%>
+                        <%=render 'shared/comments',comments:post.comments%>
+                    </div>
+                <%end%>
```

Comment:
- [ ] Kindly note that each post should link to a user post show page which displays the post with all comments. 

- [ ] Please try to add all the necessary links required to navigate from the user index page to the user show page, to the user post index page, and to the user post show page. As your app is right now, you do not have any links to navigate to different pages. I suggest you visit the project requirement again and try fixing these issues.

Reply:
The previous comment contradicts this one.
In order to display only the recent comments in the posts/index.html.erb file it has to be altered, yet in this comment, you ask me to show all comments for the same file.

Reply:
For the post show page, please show a post with all its comments as seen the wireframe.

For the post index page, please show all posts with its 5 most recent comments.

Hope it's clear now?